docs/upgrading.md

```diff
@@ -56,6 +56,21 @@ The `className` prop is unchanged and will still be passed to the underlying `<i
 
 See the [documentation](https://nextjs.org/docs/basic-features/image-optimization#styling) for more info.
 
+### Next.js' HMR connection now uses a WebSocket
+
+Previously Next.js used a [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events) connection to receive HMR events but now uses a WebSocket connection.
+
+In some cases when proxying requests to Next.js' dev server you will need to ensure the upgrade request is handled correctly, for example, in `nginx` you would need to add the below configuration:
+
+```
```

Comment:
Do we want to add any highlighting? IIRC it looks strange if you don't specify any on .org

Reply:
Looks like nginx is a supported type so added that 👍